packages/overlays/test/DropdownConfig.test.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from '@open-wc/testing';
+// import { OverlayController } from '../index.js';
+// import { withDropdownConfig } from '../src/DropdownConfig.js';
+
+const OverlayController = null; // wrapper around Global and LocalOverlayController
+
```

Comment:
Not sure if this really needs its own file? Good idea though to test the config feature and its options all to be working.

Reply:
Mainly serves as an example of how to implement Responsive Overlays with config objects instead of controllers.
Not finished yet...